chalk-ir/src/interner.rs

```diff
@@ -227,6 +231,13 @@ pub trait Interner: Debug + Copy + Eq + Ord + Hash {
         None
     }
 
+    /// Prints the debug representation of a function-def-id. To get good
+    /// results, this requires inspecting TLS, and is difficult to
```

Comment:
This isn't a comment *this* PR exactly (since this just matches other functions). But I honestly would rather we no mention TLS in all of these functions.

Reply:
Yeah, and it adds a lot of noise to the source, even more so since there's 25 of them. I'll just remove the part about TLS.